docs/writing_a_simple_publisher_and_subscriber.md

```diff
@@ -0,0 +1,479 @@
+# Writing a simple publisher and subscriber (Rust)
+* Goal: Create and run a publisher and subscriber node using Rust.
+* Tutorial level: Beginner
+* Time: 20 minutes
+<details><summary>Background</summary>
+
+In this tutorial you will create a pair of 
+[nodes](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Nodes/Understanding-ROS2-Nodes.html) that pass information to each other via a 
+[topic](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics.html) in the form of string messages. The example used here is a simple "talker" and "listener" system; one node publishes data and the other subscribes to the topic to receive that data.
+
+Since Rust doesn't have inheritance, it's not possible to inherit from `Node` as is common practice in [`rclcpp`](https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Writing-A-Simple-Cpp-Publisher-And-Subscriber.html) or [`rclpy`](https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Writing-A-Simple-Py-Publisher-And-Subscriber.html).
```

Comment:
I think this section would be more appropriate right before we instruct the user to define a node. Its out of place up here.

Reply:
To be honest: I don't see the point in rebuilding the whole text now, after so many changes have already been made. I just want to finalise or finish the whole thing. I've literally been sitting on this pull request for months and I just don't want any more.